scalafix-core/src/main/scala/scalafix/syntax/package.scala

```diff
@@ -71,6 +71,14 @@ package object syntax {
 
   implicit class XtensionToken(token: Token) {
     def posTuple: (Int, Int) = token.start -> token.end
+    // copy pasted from:
+    // github.com/scalameta/scalafmt/blob/master/scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenOps.scala
+    def hash: Long = {
```

Comment:
Can we make this `private [scalafix]`?

Which reminds me I should setup MiMa to keep track of the public api.